integration_test/third_party_apps_data/applications/couchbase/metadata.yaml

```diff
@@ -0,0 +1,73 @@
+app_url: "https://www.couchbase.com/"
+short_name: couchbase
+long_name: Couchbase
+logo_path: /stackdriver/images/couchbase.png # supplied by google technical writer
+description: |-
+  The Couchbase integration collects bucket metrics such as operations, memory usage, and ejections. The integration collects metrics from the prometheus server exposed on a node. 
+minimum_supported_agent_version:
+  metrics: 2.17.0
+  logging: 2.17.0
```

Comment:
I think i've said mutiple times before... this don't need be set manually....
We can predict which release tag this integration will go with...

Reply:
Ahh I apologize, I'll keep this in mind for the future. Truly sorry about that!